src/main/java/core/basesyntax/service/operation/SupplyOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+public class SupplyOperation implements OperationHandler {
+    @Override
+    public int getQuantity(int defaultQuantity, int fruitQuantity) {
+        return defaultQuantity + fruitQuantity;
```

Comment:
The parameters `defaultQuantity` and `fruitQuantity` could be named more clearly. For instance, `currentQuantity` and `additionalQuantity` might better convey their purpose in the context of a supply operation.